demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -181,35 +181,19 @@ export class DemoMeetingApp {
     }
   }
 
-  getAudioInputDevice = async (): Promise<MediaDeviceInfo> => {
-    const audioInputDevices = await this.deviceController.listAudioInputDevices();
+  getAudioInputDeviceID = async (): Promise<string> => {
```

Comment:
These functions no longer needs to return `Promise`s — they're just DOM lookups.